src/index.html

```diff
@@ -6,14 +6,463 @@
+        </li>
+        <li>
+          <a
+            href="#about"
+            class="menu__link"
+          >
+            About
+          </a>
+        </li>
+      </ul>
+    </div>
+
+    <main class="main">
+      <h2 class="main__text">
+        if you choose the gifts of nature and care about
+        <br />
+        it’s - scroll down
+      </h2>
+
+      <section
+        class="section"
+        id="about"
+      >
+        <h3 class="section__title">about our brand</h3>
+
+        <p class="section__text">
+          Our brand cosmetics are made from natural ingredients and we are
+          promoting them to the masses. Shampoo, hair conditioner, soap - in
+          recycled paper. Creams, oils, washing products in glass containers
+          with recycled paper labels. No synthetics and microplastics in
+          cosmetics. And the packaging from it can be returned for recycling or
+          reused.
+        </p>
+
+        <a
+          href="#address"
+          class="button--type2 section__button--type2"
+        >
+          LEARN MORE
+        </a>
+      </section>
+
+      <section
+        class="shop"
+        id="shop"
+      >
+        <h3 class="shop__title">shop</h3>
+
+        <nav class="shop__navigation">
+          <a
+            href="#face"
+            class="button--type3"
+          >
+            face
+          </a>
+          <a
+            href="#body"
+            class="button--type3"
+          >
+            body
+          </a>
+          <a
+            href="#hair"
+            class="button--type3"
+          >
+            hair
+          </a>
+          <a
+            href="#candles"
+            class="button--type3"
+          >
+            candles
+          </a>
+        </nav>
+
+        <article class="shop-card">
+          <img
+            src="./images/face.png"
+            alt="face"
+            class="shop-card__picture"
+          />
+
+          <h4 class="shop-card__title">Hydrophilic oil</h4>
+          <p class="shop-card__price">160 UAH</p>
+        </article>
+
+        <a
+          href="#address"
+          class="section__button--type1 button--type1"
+        >
+          all products
+        </a>
+      </section>
+
+      <section class="section">
+        <h3 class="section__title">about the creators</h3>
+
+        <p class="section__text">
+          My name is Victoria, I’m a master at candle making and natural
+          cosmetics. In the past, a cynologist, was engaged in herbs, made
+          layouts on the Tarot.
+        </p>
+
+        <p class="section__text">
+          I am also a volunteer to save wild birds in distress, a consultant to
+          the Phoenix Wildlife Rehabilitation Center, a member of the Society
+          for the Conservation of Birds of Ukraine with the support of Birdlife
+          International.
+        </p>
+
+        <a
+          href="#address"
+          class="button--type2 section__button--type2"
```

Comment:
same problem with BEM (https://mate-academy.github.io/fe-program/css/typical-bem-mistakes-en#using-a-modifier-without-the-belonging-class)